Sunray: update document and migrate Python script to Python3 (#230)
OpenIndiana/oi-docs: docs/handbook/sunray.md

We want to install an other version of the gnome packages as defined in userland-incorporation meta-package and relaese the version locks like shown above.
</div>

## Install on current Hipster (2022)

> If you have installed new Hipster, it is necessary to install GNOME packages for GDM and his greeter-session. For user session it is fine to select 'Mate Session', so it is not necessary any more to install other packages needed by Gnome Session (only Gnome packages which Mate session still used).

Reviewer: ```suggestion
If you have installed new Hipster, it is necessary to install GNOME packages for GDM and its greeter-session. For user session it is fine to select 'Mate Session', it is no longer necessary to install other packages needed by Gnome Session (only Gnome packages which Mate session still uses).
```

The version locks of the gnome packages have to relaese like decribed in [update older Hipster](#update-older-hipster).
</div>

### Install from Hipster (2019)

> This is outdated and not necessary anymore. But it keeped here if some one still like to run Gnome-Session on Hipster.

Reviewer: ```suggestion
This is outdated and not necessary anymore. But it is kept here in case someone would still like to run Gnome-Session on Hipster.
```

If you installed new Hipster, it is necessary to install GNOME packages before they were obsoleted from Hipster.

Last known working OpenIndiana GNOME packages versions are the following:

```
gnome-desktop@2.32.1
gnome-menus@2.30.5
gnome-panel@2.32.1
gnome-session@2.32.1
gnome-settings-daemon@2.32.1
gdm@2.30.7
metacity@2.30.3
libgnomekbd@2.32.0
libwnck@2.31.0
desktop/screensaver@5.42
```

## XScreenSaver

Latest Hipster delivers XScreenSaver only in 64-bit. The SunRay PAM module
are shipped as 32-bit only so unlocking works only with 32-bit XScreenSaver.
That's why we need the XScreenSaver package with 32-bit bins from <http://pkg.toc.de/sunray/>.

> It is more comfortable to use mate-screensaver. For PAM is only the modul mate-screensaver-pam-helper neededi. This is not an shared object instead an executable and can so shipped in 32bit independent from the other commands.

Reviewer: ```suggestion
It is more comfortable to use mate-screensaver. For PAM is only the module mate-screensaver-pam-helper needed. This is not an shared object, instead it is an executable and can be shipped in 32bit independent from the other commands.
```
